osu.Framework/Input/Handlers/Joystick/JoystickHandler.cs

```diff
@@ -10,7 +11,12 @@ namespace osu.Framework.Input.Handlers.Joystick
 {
     public class JoystickHandler : InputHandler
     {
-        private const float deadzone_threshold = 0.075f;
+        public BindableFloat DeadzoneThreshold { get; } = new BindableFloat(0.075f)
```

Comment:
```suggestion
        public BindableFloat DeadzoneThreshold { get; } = new BindableFloat(0.1f)
```

Sorry to request another change, but while we're here I think the default value could do with a slight bump.